How_to_Release.md

```diff
@@ -0,0 +1,23 @@
+# Creating a new release
+
+This is a guide how to create a new ruff release
+
+1. Update the version with `rg 0.0.269 --files-with-matches | xargs sed -i 's/0.0.269/0.0.270/g'`
+1. Update [BREAKING_CHANGES.md](BREAKING_CHANGES.md)
+1. Create a PR with the version and BREAKING_CHANGES.md updated
+1. Merge the PR
+1. Run the release workflow with the tag (without starting `v`) as input. Make sure main has your
```

Comment:
Should it be possible to provide an optional commit to the release workflow that it then uses as the version to publish? This could be useful to retry a release after something has been merged. But we could also support this by reverting all changes and then releasing

Reply:
This doesn't work like i would like it to, i reverted the change.

The main problem with this is that we run the workflow on code from a different commit than the release.yml is from, and this could become really strange. I'd instead prefer to merge new changes to main and then run on those.